.circleci/config.yml

```diff
@@ -163,28 +152,27 @@ jobs:
       - check_release_chaneglog_update_branch
       - merge_target_branch
       - install_chrome
-      - attach_workspace:
-          at: /home/circleci/
+      - install_dependencies
       - run:
           name: Run e2e publication test
           command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="publication" --prod_env
+            python -m scripts.run_e2e_tests --deparallelize_terser --suite="publication" --prod_env
       - run:
           name: Run e2e library test
           command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="library" --prod_env
+            python -m scripts.run_e2e_tests --skip-build --skip-install --deparallelize_terser --suite="library" --prod_env
       - run:
           name: Run e2e classroomPage test
           command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="classroomPage" --prod_env
+            python -m scripts.run_e2e_tests --skip-build --skip-install --deparallelize_terser --suite="classroomPage" --prod_env
       - run:
           name: Run e2e collections test
           command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="collections" --prod_env
+            python -m scripts.run_e2e_tests --skip-build --skip-install --deparallelize_terser --suite="collections" --prod_env
       - run:
           name: Run e2e accessibility page test
           command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="accessibility" --prod_env
+            python -m scripts.run_e2e_tests --skip-build --skip-install --deparallelize_terser --suite="accessibility" --prod_env
```

Comment:
Why is the ``deparallelize_terser`` setting needed now?

Reply:
I saw few webpack related failures for tests that used the "prod_env" flag (seen [here](https://app.circleci.com/pipelines/github/kevintab95/oppia/245/workflows/f97722ea-a608-4516-95f3-4580b7e463c0/jobs/2250)) after I got rid of the "persist to workspace" step. Adding this flag seems to fix the error.